core/templates/dev/head/pages/Base.ts

```diff
@@ -16,6 +16,7 @@ require('domain/sidebar/SidebarStatusService.ts');
 require('domain/utilities/UrlInterpolationService.ts');
 require('services/AlertsService.ts');
 require('services/contextual/UrlService.ts');
+require('services/CsrfService.ts');
```

Comment:
The correct order, please. Move it before `require('services/contextual/UrlService.ts');`